### DISCUSSION TITLE
Recommended cameras in Europe

### DISCUSSION THREAD

**Discussion by lkaino** [blakeblackshear/frigate#6234] [General]
The Amcrest camera seems awesome, but is not available in Europe.

Does anyone know any good cameras at similar price range available in Europe? Dahua is bit more expensive and there’s million different models available. 

Are there any cameras available on Aliexpress that don’t upload video/images to China?

**Comment by PovilasID** [comment]
a) Look for freight forwarding services. You will be able to find things like 'cheap shipping from US/amazon'. Their business model is simple they take a bunch of orders and ship them together drastically reducing the price. Now... some of them are sketchy so look for reviews and look for local companies.
b) Look at your local used markets. I have a couple of notifications trained on used sites and I got sweet deal on Dahua cam.
c) Chine's cameras do not send video... if you do not let them. It takes a bit of custom knowhow but you can ban devices on your network from accessing internet. Without ability to connect to remote servers they will not be able to send anything. That said... cameras from Ali can have issues... fragile plastic construction... lower resolution than actually advertised.
